.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "extends": "airbnb",
+  "plugins": [
+    "react",
+    "jsx-a11y",
+    "import"
+  ],
+  "rules": {
+    "import/no-unresolved": "off",
+    "import/no-extraneous-dependencies": "off",
+    "import/extensions": "off",
+    "no-console": "off",
+  },
+  "globals": {
+    "document": true,
+    "window": true,
```

Comment:
Not sure about this. What do you want to express with it? The code in this repository is half node (webpack) and half browser (`hello_*.js`).

Maybe it's good to define both envs:

```
{
    "env": {
        "browser": true,
        "node": true,
    }
}
```

http://eslint.org/docs/user-guide/configuring.html#specifying-environments